corehq/apps/hqwebapp/templates/hqwebapp/partials/ko_select_toggle.html

```diff
@@ -6,7 +6,12 @@
                                attr: {selected: $data.selected}"></option>
     </select>
     <div class="btn-group-separated" data-bind="foreach: options">
-      <button class="btn btn-default"
-              data-bind="text: $data.text, css: {active: $data.selected}, click: click"></button>
+      <button class="btn btn-default btn-disabled"
```

Comment:
Why is `btn-disabled` always being added? I'd think it would only be added if $parent is disabled.